src/main/scala/lenthall/spray/WrappedRoute.scala

```diff
@@ -1,31 +0,0 @@
-package lenthall.spray
-
-import spray.routing._
-import spray.routing.directives.PathDirectives
-
-object WrappedRoute {
```

Comment:
Nope, I"m one step ahead of you :) That's not going to happen as per a discussion w/ the TLs overseeing thurloe.